test/e2e/publicviewer/spacebindingrequest_test.go

```diff
@@ -0,0 +1,90 @@
+package publicviewer_test
+
+import (
+	"sort"
+	"testing"
+
+	"github.com/gofrs/uuid"
+	"github.com/stretchr/testify/require"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+
+	testspace "github.com/codeready-toolchain/toolchain-common/pkg/test/space"
+	spacebindingrequesttestcommon "github.com/codeready-toolchain/toolchain-common/pkg/test/spacebindingrequest"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	testsupportspace "github.com/codeready-toolchain/toolchain-e2e/testsupport/space"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/spacebinding"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+)
+
+func NewSpaceBindingRequest(
```

Comment:
just curious - was there any problem with reusing the existing function from https://github.com/codeready-toolchain/toolchain-e2e/blob/82dc979b1fafe3bf61792736cad45b71baead750/test/e2e/parallel/spacebindingrequest_test.go#L231

Plus as mentioned in my other comment you probably don't need this function, but maybe I'm missing something.

Reply:
done in c4660e0332eb71ee3bec7645110ea5df8869ccb9